compiler/rustc_passes/messages.ftl

```diff
@@ -427,6 +427,10 @@ passes_link_section =
     .warn = {-passes_previously_accepted}
     .label = not a function or static
 
+passes_linkage =
+    attribute should be applied to a free function, impl method or static, foreign static
```

Comment:
```suggestion
    attribute should be applied to a function or a static
```

Reply:
I changed it to "to a function or static" to align with the other error messages.